config/creds/README.creds

```diff
@@ -0,0 +1,21 @@
+This directory will hold credentials files on the build host.
+
+All JSON files in this directory are ignored via gitignore(5) to reduce the
+risk that credentials accentally get commit.
+
+The following files are currently used:
+
+* gce.json
+
+  A service account JSON file for the Google Cloud Platform. This file can be
+  generated as describe here:
```

Comment:
We call it `gce.json` but say `Google Cloud Platform`.  Which one should it be for consistency?